build/docker/Dockerfile.sites-generic

```diff
@@ -61,6 +62,9 @@ RUN yarn install --frozen-lockfile
 FROM deps AS test
 
 ARG SITE
+ARG CLOUDINARY_CLOUD_NAME
+ARG FILE_SERVICE
+ARG LISTINGS_QUERY
 
 # We need to have this nested 2 layers deep due to hardcoded paths in package.json
```

Comment:
That's frustrating to have to have it twice!

Reply:
Please explain what you mean.

Reply:
I think it's just around the repetition of the `ARG`s, which _is_ annoying but it's also kind of a docker thing.  Fortunately all of those are going away soon

Reply:
I believe the docker thing you're both referring to is that the env vars need to be declared if they are used in a given stage.